server/opts.go

```diff
@@ -1714,7 +1761,12 @@ func parseJetStream(v interface{}, opts *Options, errors *[]error, warnings *[]e
 				opts.JetStreamMaxMemory = mv.(int64)
 				opts.maxMemSet = true
 			case "max_file_store", "max_file":
-				opts.JetStreamMaxStore = mv.(int64)
+				// need to define error to avoid non-name on left side of :=
```

Comment:
Yes, or you could do (assuming the function is renamed)
```
sz, err := getStorageSize(mv)
if err != nil {
 ...
}
opts.JetStreamMaxStore = sz
..
```

Reply:
yeah switched to this method.